src/hooks.tsx

```diff
@@ -90,7 +99,9 @@ const createReducer = <T extends DataType>() => (
       const columnCopy = state.columns.map(column => {
         // if the row was found
         if (action.columnName === column.name) {
-          isAscending = column.sorted.asc;
+          // if it's undefined, start by setting to ascending, otherwise toggle
+          isAscending =
+            column.sorted.asc === undefined ? true : !column.sorted.asc;
```

Comment:
If `column.sorted.asc` is already set to true, won't this set it to false? Seems like we should define `column.sorted.asc` as true/false earlier instead of letting it be undefined.

Reply:
Yeah because this is the toggle sort action (like when you click the header), so we want it to go from true -> false and false->true.

In the case that it's undefined (the default), we just want to start with ascending order, if that makes sense.  Undefined marks the state before any sorting has happened.  If you think this is more confusing I can set it to true/false only again.

Reply:
Ah my mistake, I didn't realize we were in the `TOGGLE_SORT` case. I generally think it's better to not leave things undefined, but it's your call.

Reply:
Yeah I generally agree.  I guess I could make it null | boolean which might be slightly better.